MidCamp/Hatter
# change banner for 2018

Update the banner with 2018 and the dates and change the background to be something with less white guys. 

#### Includes
* https://github.com/orgs/MidCamp/projects/1#card-4864930
* https://github.com/MidCamp/midcamp/issues/44

#### Testing instructions
* checkout branch
* `npm run butler`
* Navigate to Homepage (http://localhost:4000/home)
* Observe the banner is using the drop background image

##### Remaining tasks
* [ ] Set card to done
* [ ] Comment (and close) issue
